src/parser/parser.yy

```diff
@@ -247,6 +247,7 @@ class GraphScanner;
 
 name_label
      : LABEL { $$ = $1; }
+     | STRING { $$ = $1; }
```

Comment:
Empty string for filed name workable?
And I think the empty filed will be conflict with `ttl_col = ""`.

Reply:
Good catch, the name_lable can't be empty.